src/app/(admin)/admin/_components/Specialist/Create.js

```diff
@@ -8,6 +8,7 @@ import { Details } from '@admin/components/Specialist/Details';
 import { Addresses } from '@admin/components/Specialist/Addresses';
 import { Services } from '@admin/components/Specialist/Services';
 import { Contacts } from '@admin/components/Specialist/Contacts';
+import { SocialLinks } from '@admin/components/Specialist/SocialLinks';
```

Comment:
🟠 could you use relative imports instead?

Reply:
aren't we suppose to use path aliases?)